src/default.jl

```diff
@@ -159,7 +159,8 @@ function defaultalg(A, b, assump::OperatorAssumptions{Bool})
                (__conditioning(assump) === OperatorCondition.IllConditioned ||
                 __conditioning(assump) === OperatorCondition.WellConditioned)
                 if length(b) <= 10
-                    DefaultAlgorithmChoice.GenericLUFactorization
+                    # DefaultAlgorithmChoice.GenericLUFactorization
+                    DefaultAlgorithmChoice.RFLUFactorization
```

Comment:
I'm not sure that's anything more than voodoo, since https://github.com/JuliaLinearAlgebra/RecursiveFactorization.jl/blob/master/src/lu.jl#L62-L69 . I don't think that uses more caches or anything?

Reply:
RFLU implementation here bypasses that to directly call https://github.com/JuliaLinearAlgebra/RecursiveFactorization.jl/blob/425e43622e8bb4a9e0a00f843791ab40df2f9466/src/lu.jl#L89